server/checker/namespace_checker.go

```diff
@@ -35,10 +38,11 @@ type NamespaceChecker struct {
 // NewNamespaceChecker creates a namespace checker.
 func NewNamespaceChecker(cluster schedule.Cluster, classifier namespace.Classifier) *NamespaceChecker {
 	filters := []filter.Filter{
-		filter.StoreStateFilter{MoveRegion: true},
+		filter.StoreStateFilter{ActionScope: namespaceCheckerName, MoveRegion: true},
```

Comment:
Does it seemed to be better to remove `name` in `namespaceCheckerName`? And other similar places.